BobClient/Qoollo.BobClient/BobNodeClient.cs

```diff
@@ -518,7 +519,171 @@ public Task<byte[]> GetAsync(ulong key)
             return GetAsync(key, false, new CancellationToken());
         }
 
+        /// <summary>
+        /// Checks data presented in Bob
+        /// </summary>
+        /// <param name="keys">Keys array</param>
+        /// <param name="fullGet">Try read data from sup nodes</param>
+        /// <param name="token">Cancellation token</param>
+        /// /// <returns>Operation result</returns>
+        /// <exception cref="ObjectDisposedException">Client was closed</exception>
+        /// <exception cref="TimeoutException">Timeout reached</exception>
+        /// <exception cref="OperationCanceledException">Operation was cancelled</exception>
+        /// <exception cref="BobOperationException">Other operation errors</exception>
+        /// <exception cref="ArgumentNullException">keys is null</exception>
+        public bool[] Exists(ulong[] keys, bool fullGet, CancellationToken token)
+        {
+            if (_isDisposed)
+                throw new ObjectDisposedException(GetType().Name);
+
+            if (keys == null)

```

Comment:
Checking arguments is usually performed before checking the state. Move this check right before `if (_isDisposed)`